apps/meteor/app/e2e/client/rocketchat.e2e.ts

```diff
@@ -507,6 +517,65 @@ class E2E extends Emitter {
 
 		return message;
 	}
+
+	fetchUsersWithWaitingKeys() {
+		return sdk.rest.get('/v1/e2e.fetchUsersWaitingForGroupKey');
```

Comment:
I'm wondering if we could do this on the client side only. For the rooms I am a participant, the field telling if users are waiting for the queue should be available on the client side, isn't it? This way we can save this call and not add an extra endpoint.

Reply:
I think we can keep your implementation here since it could be a problem having this field available on the client-side. Let's make sure this is not available there.